### DISCUSSION TITLE
2006/04/27/%E6%8B%89%E6%96%AF%E7%BB%B4%E5%8A%A0%E6%96%AF%E6%B8%B8%E8%AE%B0

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#82] [Announcements]
# 2006/04/27/%E6%8B%89%E6%96%AF%E7%BB%B4%E5%8A%A0%E6%96%AF%E6%B8%B8%E8%AE%B0

周末刚去拉斯维加斯玩了一圈。

https://blog.qizhen.xyz/2006/04/27/%E6%8B%89%E6%96%AF%E7%BB%B4%E5%8A%A0%E6%96%AF%E6%B8%B8%E8%AE%B0

**Comment by ruanqizhen** [comment]
之前博客下的留言：

nicy
2010年03月1日 03:09 编辑
拉斯维加斯~~~ 疯狂的城市~

回复
花朵旖旎
2006年06月9日 11:27 编辑
旅行分为扫盲班和深度游，你就当作是先扫扫盲，然后下次去深度游吧。细细！

回复
666666
2006年04月28日 08:43 编辑
你好来你家串门了,快到五一了,提前祝你快乐,希望咱们能成为好邻居,美好的时光咱们一起分享:)
